core-ui/src/main/res/drawable/ic_onboarding_settings.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="32dp"
+    android:height="32dp"
+    android:viewportWidth="32"
+    android:viewportHeight="32">
+  <path
+      android:pathData="M15.25,5.75C15.25,5.336 15.586,5 16,5C16.414,5 16.75,5.336 16.75,5.75V7.533C17.755,7.621 18.711,7.883 19.585,8.291L20.476,6.748C20.683,6.39 21.141,6.267 21.5,6.474C21.796,6.645 21.932,6.988 21.853,7.304C21.837,7.371 21.81,7.436 21.774,7.499L20.883,9.042C21.689,9.608 22.392,10.311 22.958,11.117L24.502,10.225C24.861,10.018 25.319,10.141 25.526,10.5C25.733,10.859 25.611,11.317 25.252,11.524L23.709,12.415C24.117,13.289 24.379,14.245 24.467,15.25H26.25C26.664,15.25 27,15.586 27,16C27,16.414 26.664,16.75 26.25,16.75H24.467C24.379,17.755 24.117,18.711 23.709,19.585L25.252,20.476C25.611,20.683 25.733,21.141 25.526,21.5C25.319,21.859 24.861,21.982 24.502,21.775L22.958,20.883C22.392,21.689 21.689,22.392 20.883,22.958L21.774,24.502C21.852,24.636 21.883,24.785 21.873,24.929C21.86,25.115 21.778,25.293 21.637,25.425C21.596,25.463 21.551,25.497 21.5,25.526C21.141,25.733 20.683,25.61 20.476,25.252L19.585,23.709C18.711,24.117 17.755,24.379 16.75,24.467V26.25C16.75,26.664 16.414,27 16,27C15.586,27 15.25,26.664 15.25,26.25V24.467C14.245,24.379 13.29,24.117 12.415,23.709L11.524,25.252C11.317,25.611 10.859,25.733 10.5,25.526C10.141,25.319 10.018,24.861 10.226,24.502L11.117,22.958C10.311,22.392 9.608,21.689 9.042,20.883L7.498,21.774C7.139,21.982 6.681,21.859 6.474,21.5C6.267,21.141 6.389,20.683 6.748,20.475L8.291,19.585C7.883,18.711 7.621,17.755 7.533,16.75H5.75C5.724,16.75 5.699,16.749 5.673,16.746C5.295,16.708 5,16.388 5,16C5,15.586 5.336,15.25 5.75,15.25H7.533C7.621,14.245 7.883,13.29 8.291,12.415L6.748,11.525C6.389,11.318 6.267,10.859 6.474,10.5C6.681,10.141 7.139,10.019 7.498,10.226L9.042,11.117C9.608,10.311 10.311,9.608 11.117,9.042L10.226,7.498C10.018,7.139 10.141,6.681 10.5,6.474C10.859,6.267 11.317,6.389 11.524,6.748L12.415,8.291C13.29,7.883 14.245,7.621 15.25,7.533V5.75ZM9.04,16.75C9.414,20.263 12.387,23 16,23C17.008,23 17.966,22.787 18.831,22.404L15.567,16.75H9.04ZM15.567,15.25H9.04C9.414,11.737 12.387,9 16,9C17.008,9 17.966,9.213 18.831,9.596L15.567,15.25ZM20.13,21.653L16.866,16L20.13,10.347C21.87,11.621 23,13.678 23,16C23,18.322 21.87,20.379 20.13,21.653Z"
+      android:fillColor="#7B7A72"
```

Comment:
glyph_active

Reply:
this color is not themed, onboarding only has dark mode colors